### PR TITLE
Support phonon calculation in autotest through vasp and dp

### PR DESCRIPTION
**Pull request by Chengqian-Zhang**
The two parameters to be provided by the user are
1、"band_path": the path of k-points.For example: "0 0 0 0.5 0.5 0.5"
2、"supercell_matrix": Multiplicity of cell expansion.For example: [2 2 2]

### CODE REVIEW

File: dpgen/auto_test/common_prop.py [hunk 1]
```diff
@@ -187,10 +192,17 @@ def worker(work_path,
...
+    if(property_type == "phonon"):
+        supercell_matrix = loadfn(os.path.join(all_task[0],'task.json'))['supercell_matrix']
+    else:
+        supercell_matrix = None
     run_tasks = [os.path.basename(ii) for ii in all_task]
-    machine, resources, command, group_size = util.get_machine_info(mdata, inter_type)
+    machine, resources, command, group_size = util.get_machine_info(mdata, inter_type, property_type , supercell_matrix)
     api_version = mdata.get('api_version', '0.9')
+
+    print(command)
```
Comment (njzjz) [inline dpgen/auto_test/common_prop.py:205]: debug code?

**Review by wanghan-iapcm — COMMENTED** [review]
Could you please provide unittests for your PR? Thank you.

**Comment by wanghan-iapcm** [comment]
Please also update the doc.

**Review by wanghan-iapcm — COMMENTED** [review]
Why do you need to add the data files `tests/data/POSCAR.01x01x01`, `tests/data/surf.al.fcc.01x01x01/`?
Why the data in `test/dispatcher` are revised?

**Comment by wanghan-iapcm** [comment]
You may need to merge the devel to solve the issues in the UTs

File: dpgen/auto_test/common_prop.py [hunk 2]
```diff
@@ -187,9 +192,17 @@ def worker(work_path,
...
+    if(property_type == "phonon"):
+        supercell_matrix = loadfn(os.path.join(all_task[0],'task.json'))['supercell_matrix']
+    else:
+        supercell_matrix = None
     run_tasks = [os.path.basename(ii) for ii in all_task]
-    machine, resources, command, group_size = util.get_machine_info(mdata, inter_type)
+    machine, resources, command, group_size = util.get_machine_info(mdata, inter_type, property_type)
+    if('phonolammps' not in command and not inter_type in ['vasp','abacus']):
+        command = "/root/local/deepmd-kit-2.1.0/bin/phonolammps in.lammps -c POSCAR --dim %s %s %s && /root/local/deepmd-kit-2.1.0/bin/lmp -i in.lammps"%(supercell_matrix[0],supercell_matrix[1],supercell_matrix[2])
+        print(f"Warning:We done find 'phonolammps' in your command sentence . We will use '/root/local/deepmd-kit-2.1.0/bin/phonolammps in.lammps -c POSCAR --dim %s %s %s'"%(supercell_matrix[0],supercell_matrix[1],supercell_matrix[2]))
```
Comment (njzjz) [inline dpgen/auto_test/common_prop.py:205]: Hard coded file paths should not be submitted.

**Pull request closed without merging**